src/js/addons/pickers/ColorPicker.js

```diff
@@ -145,10 +164,18 @@ export default class ColorPicker extends Picker {
 
         if (colorDisc.getContext) {
             // HSV color wheel with white center
+            let diskContex = colorDisc.getContext('2d');
+            let ratio = getDevicePixelRatio(diskContex);
+            let width = this.disc.width / ratio;
+            let height = this.disc.height / ratio;
+            this.dom.colorDisc.width = width * ratio;
+            this.dom.colorDisc.height = height* ratio;
+            diskContex.scale(ratio, ratio);
+
             drawDisk(
-                colorDisc.getContext('2d'),
-                [colorDisc.width / 2, colorDisc.height / 2],
-                [colorDisc.width / 2 - 1, colorDisc.height / 2 - 1],
+                diskContex,
+                [width/2, height/2],
+                [width/2 - 1, height/2 - 1],
```

Comment:
Please add spaces around operators, generally a good practice for readability, see, for example:
- http://airbnb.io/javascript/#18.4
- http://standardjs.com/rules.html

This should be enforced by the linter.